plugins/repoclosure.py

```diff
@@ -66,6 +66,8 @@ def run(self):
             print("  unresolved deps:")
             for dep in unresolved[pkg]:
                 print("    {}".format(dep))
+        if len(unresolved) > 0:
+            self.cli.demands.success_exit_status = 1
```

Comment:
return 1 was my first idea. But dnf works another way. Return code of run func is not used.